utilization.md

```diff
@@ -161,141 +217,77 @@ Kokkos::View<double**[25]> view3D("view3D", 50, 42, 25);
 </details>
 <!--#endif-->
 
-#### Accessing elements
+#### Manage
 
-```cpp
-for (int i = 0; i < 10; i++) {
-    for (int j = 0; j < 50; ++j) {
-        view(i, j) = i + j;
-    }
-}
-```
+| Method        | Description                                                                                                           |
+|---------------|-----------------------------------------------------------------------------------------------------------------------|
+| `(i, j...)`   | Returns and sets the value                                                                                            |
```

Comment:
Access the value indexed by (i...)

Reply:
To be honest, I'm not happy by calling `(i, j...)` a method. I can follow your comment to make it consistent though.